types/stripe/index.d.ts

```diff
@@ -298,35 +290,473 @@ declare namespace Stripe {
             };
 
             /**
-             * The URL that best shows the service or product provided for this account
+             * The business type. Can be individual or company.
              */
-            business_url?: string;
+            business_type?: "individual" | "company";
 
             /**
-             * A boolean for whether or not Stripe should try to reclaim negative
-             * balances from the account holder’s bank account. See our managed
-             * account bank transfer guide for more information
+             * Information about the company or business.
+             * This field is null unless business_type is set to company.
              */
-            debit_negative_balances?: boolean;
+            company?: {
+                /**
+                 * The company’s primary address.
+                 */
+                address?: {
```

Comment:
There is a new `IAddress`, please could you use this

Reply:
:+1: Implemented